source/views.txt

```diff
@@ -125,12 +123,13 @@ Modify the Source of a View
 
 To modify the source of a view:
 
-1. Click the desired view in the left navigation.
+1. Hover over the desired view in the left navigation.
+
+#. Click the appearing :guilabel:`Ellipses (...)` button.
 
-#. At the top of the view, click :guilabel:`Modify Source`. This
-   button opens the
-   :ref:`aggregation pipeline builder <compass-agg-builder>` and
-   populates the pipeline used to create the view.
+#. In the dropdown, click :guilabel:`Modify view`. This
```

Comment:
([terminology](https://www.mongodb.com/docs/meta/style-guide/terminology/alphabetical-terms/#d))
There are a few ways you could change this. But probably "In the drop-down menu..."

Reply:
Updated!